kotsadm/web/src/components/apps/AppVersionHistoryRow.jsx

```diff
@@ -46,7 +47,15 @@ function deployButtonStatus(downstream, version, app) {
   }
 }
 
-function renderVersionAction(version, nothingToCommitDiff, app, history, deployVersion) {
+function checkIsCurrentOrPastVersion(latestVersion, version) {
```

Comment:
maybe `isLatestVersion` is a better name and flip the condition to match name, `true => false` and `false => true`

Reply:
or change it to `isEditable`?